config/globals.js

```diff
@@ -59,7 +59,7 @@ const globals = {
 	ENVIRONMENTS,
 	LOG_LEVEL: process.env.LOG_LEVEL || defaultLogLevel,
 	/** HOST=https://schul-cloud.org in bosscloud, see misuse in todo */
-	HOST: process.env.HOST || 'localhost:3100', // todo this is client url in config
+	HOST: process.env.HOST || 'http://localhost:3100', // todo this is client url in config
```

Comment:
please move this to the configuration

Reply:
:+1:  done - I have added it only to default.json and production.json . Is it ok ?